build_projects/update-dependencies/update-dependencies.ps1

```diff
@@ -17,20 +17,18 @@ if($Help)
     exit 0
 }
 
-$Architecture='x64'
-
 $RepoRoot = "$PSScriptRoot\..\.."
 $AppPath = "$PSScriptRoot"
 
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows\$Architecture"
+    $env:DOTNET_INSTALL_DIR="$RepoRoot\.dotnet_stage0\Windows"
```

Comment:
Why would we want to build two different architectures without cleaning in between? That leaves build artifacts and changes that should not be there and could have an adverse affect on the build...we should not leave any artifacts behind that could change the build in any way between builds